### DISCUSSION TITLE
Load items during drag (i.e. infinite scroll support)

### DISCUSSION THREAD

**Discussion by watlandc** [clauderic/dnd-kit#103] [Q&A] [answered]
This kit looks fantastic – excellent work!

With sortable lists, is it possible to load additional items during drag (i.e. infinite scroll)?

Even if it's not possible to load while dragging, could infinite scroll be used (while scrolling, not dragging) without breaking the implementation?

Our app has long lists that require infinite scroll and dnd.

**Comment by clauderic** [comment, marked as answer]
Haven't tried this yet, but off the top of my head I think this use-case should work perfectly fine, it's somewhat similar to a virtualized list, the main difference compared to a virtualized list is that items get added to the `SortableContext` as you're scrolling

**Reply by watlandc** [reply]
👍 Sounds promising. We will update findings if it doesn't work as expected.
